snapshot.md

```diff
@@ -3,47 +3,165 @@
 layout: snapshot
 keywords:
 comments: false
-
 # Hero section
-title: Mainnet snapshots
-description: 
-
+title: Tezos snapshots for ${NETWORK}
+description:
 # Author box
 author:
-    title: Oxhead Alpha
-    title_url: 'https://oxheadalpha.com'
-    external_url: true
-    description: Oxhead Alpha is a tezos core development company
-
+  title: Brought to you by Oxhead Alpha
+  title_url: "https://medium.com/the-aleph"
+  external_url: true
+  description: A Tezos core development company, providing common goods for the Tezos ecosystem. <a href="https://medium.com/the-aleph" target="_blank">Learn more</a>.
 # Micro navigation
 micro_nav: true
-
 # Page navigation
 page_nav:
-    home:
-        content: Previous page
-        url: 'https://xtz-shots.io/index.html'
+  home:
+    content: Previous page
+    url: https://xtz-shots.io/index.html
 ---
 
-## Tezos snapshot for ${TEZOS_NETWORK}
+{% assign network_substring = site.data.tezos_metadata.network | remove: "net" %}
+{% capture domain_name %}https://{{ site.data.tezos_metadata.network }}.xtz-shots.io{% endcapture %}
```

Comment:
Just curious if this commit was necessary? https://github.com/oxheadalpha/xtz-shots-website/pull/2/commits/c5410e39497ec4bd43cc2ab63b1c11cd95b395f2
I made the xtz-shots domain a variable in the _config.yaml
```yaml
domain_name: xtz-shots.io
```

Reply:
Ah it wasn't carrying over since we're just curling the variable so I hard coded it. Can I add it to the config in snapshot_maker?

Reply:
I'm probably missing something but, if we are using remote theme and the variable is already defined in the themes config, why does it need to be defined anywhere else or hard coded? 

Will the curled file when bundled not read the theme's config file for data?

Reply:
I had the same question, but alas the theme's config does not appear to be read.

Reply:
Ok thanks. Weird I was so sure I got it to work 🤷